src/Java.Interop.Localization/Java.Interop.Localization.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>$(MSBuildRequiredTargetFramework)</TargetFramework>
```

Comment:
Oh, *this* is where `$(MSBuildRequiredTargetFramework)` comes in?  So it's not a "standard MSBuild-ism", but is instead something we use?

I think we should use a `JI*` prefix so that I don't get confused. :-) Perhaps `$(JITargetFrameworkForMSBuildTasks)`?

Reply:
Let's get @pjcollins opinion.  It's modeled after the `$(DotNetTargetFramework)` that we use everywhere else.  

We could also just hardcode `netstandard2.0` back in the project files.  I changed it because every time I see `netstandard2.0` I want to upgrade it and I forget we can't because it's used by our MSBuild tasks.  But I guess it will never change, so we could just leave it hardcoded in the project files.

Reply:
I don't have a strong opinion either way, I think `netstandard2.0` is fine everywhere, or perhaps a rename to something like `$(BuildTaskTargetFramework)` or `$(JIBuildTaskTargetFramework)` would be good.

Reply:
Removed in favor of just using `netstandard2.0`.